libkbfs/keybase_service_base.go

```diff
@@ -292,7 +292,14 @@ func (k *KeybaseServiceBase) Identify(ctx context.Context, assertion, reason str
 		Reason:        keybase1.IdentifyReason{Reason: reason},
 	}
 	res, err := k.identifyClient.Identify2(ctx, arg)
-	if err != nil {
+	// Identify2 still returns the user-plus-keys data, even if it
```

Comment:
Okay, I think i was confused by the comment saying it returns the "user-plus-keys" data. Maybe add " (with no keys)"?

So a couple of questions:

1) Does the tracker popup still show for these sigchain-less users even though we get an error? Otherwise I'd be worried that one could sneak these users in somehow.
2) So i guess this is similar to SBS in that once the user gets a sigchain and adds some keys, one of the other users just rekeys for those keys, right?